.github/workflows/truffle-ruby-test.yml

```diff
@@ -0,0 +1,109 @@
+name: build-with-truffleruby-head
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 14 * * *"
+
+jobs:
+  ruby-versions:
+    uses: ruby/actions/.github/workflows/ruby_versions.yml@master
+    with:
+      engine: cruby-truffleruby
+      min_version: 2.6
+
+  reline:
+    needs: ruby-versions
+    name: >-
+      reline ${{ matrix.os }} ${{ matrix.ruby }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [truffleruby-head]
```

Comment:
`truffleruby-head` was originally excluded, but since it can now be easily stopped, it has been set to `truffleruby-head`.